### PR TITLE
Multiline helper

### LINKED ISSUE #18: Error message clipped
Hey, 
I found that error message under input filed will be clipped if text is too long. 
Could you please make it configurable to be able to show in multiline? 
screenshot 2017-05-31 17 06 33

### LINKED ISSUE #59: Multiline helper(title) text
Awesome work.

Is it possible to have a multiline helper text?

### PR DESCRIPTION
**Pull request by AlexisLeon**
For title and error text.
Has crossfade animation and does not affect character counter layout.
Fix #59
Fix #18

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ class Example extends Component {
  errorColor            | Text field color for errored state          |   String | rgb(213, 0, 0)
  disabledLineType      | Text field line type in disabled state      |   String | dotted
  animationDuration     | Text field animation duration in ms         |   Number | 225
+ helperNumberOfLines   | Text field helper number or lines           |   Number | -
```
Comment (adelegard) [inline readme.md]: It would be nice if you didn't have to know this in advance, but for the component to just handle text of any length.

**Comment by petrogad** [comment]
While the solution is nice, is there a way to avoid requiring the knowledge up front of the number of lines something might take?

Device width can vary and what might be 1 line for one might be 3 for another.

File: readme.md (hunk 1 above)
Comment (AlexisLeon) [reply readme.md]: Handle text of any length requires to calculate the length based on styling and container size. This is made via callback to native side which doesn’t match the component lifecycle.

Comment (adelegard) [reply readme.md]: Knowing the number of lines you require is just not realistic though. On an iphone X it may be one line, but an iphone 4 it may be 3.

**Comment by Billy-** [comment]
I agree with @adelegard's comment; hard coding the number of lines is simply not possible as it may vary  from device to device, may vary with orientation, not to mention the text may be entirely dynamic.

Would it be possible to use a solution involving `onLayout` on the container for the helper text, and keeping it's height in state?

**Comment by jordanmkoncz** [comment]
Yep I also agree with @adelegard and @Billy-, I think a solution where you have to specify the number of lines is unfortunately not actually useful, because in most cases you won't know ahead of time how many lines the text will be when it's rendered.

I think there's two possible approaches to this issue:

1) Dynamically determine/calculate the size of the rendered text content and then use this measurement where required. This could possibly be done using `onLayout`, or a library like https://github.com/airamrguez/react-native-measure-text.

2) Change how the title/error text content is rendered so that the layout will just work correctly regardless of how many lines it is, i.e. don't use absolute positioning. Unsure whether this approach is actually possible, I think at the very least you may have to sacrifice certain animations if you went with this approach.

**Comment by AlexisLeon** [comment]
@adelegard @Billy- @jordanmkoncz Agree; What about this new approach..
As render is made from top to bottom, helper can actually be rendered without the numberOfLines prop and without sacrificing animations

**Comment by pascaloliv** [comment]
PR looks good. :+1: Any news on the review ?

**Comment by n4kz** [comment]
Thanks for PR and sorry for delayed reply. I've released multiline helper support in `0.14.0` with approach different to one proposed here.

**Pull request closed without merging**